x/msgfees/spec/02_state.md

```diff
@@ -0,0 +1,23 @@
+<!--
+order: 2
+-->
+
+# State
+`MsgFee` is the core of what gets stored on the blockchain
+ it consists of two parts
+ 1. the msg type url, i.e. /cosmos.bank.v1beta1.MsgSend
+ 2. minimum additional fees(can be of any denom)
+ 
+```protobuf
+message MsgFee {
+string msg_type_url = 1;
+// additional_fee can pay in any Coin( basically a Denom and Amount, Amount can be zero)
+cosmos.base.v1beta1.Coin additional_fee = 2 [
+(gogoproto.nullable)     = false,
+(gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins",
+(gogoproto.moretags)     = "yaml:\"additional_fee\""
+];
+}
+```
```

Comment:
There should be a link to the message in the proto file in here somewhere. See other state docs for examples.